---a/net/minecraft/entity/ai/EntityAIBeg.java
+++b/net/minecraft/entity/ai/EntityAIBeg.java
@@ -6,8 +6,8 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.world.World;
 
-public class EntityAIBeg extends EntityAIBase
-{
+public class EntityAIBeg extends EntityAIBase {
+
     private EntityWolf theWolf;
     private EntityPlayer thePlayer;
     private World worldObject;
@@ -15,47 +15,39 @@
     private int field_75384_e;
     private static final String __OBFID = "CL_00001576";
 
-    public EntityAIBeg(EntityWolf p_i1617_1_, float p_i1617_2_)
-    {
+    public EntityAIBeg(EntityWolf p_i1617_1_, float p_i1617_2_) {
         this.theWolf = p_i1617_1_;
         this.worldObject = p_i1617_1_.worldObj;
         this.minPlayerDistance = p_i1617_2_;
         this.setMutexBits(2);
     }
 
-    public boolean shouldExecute()
-    {
-        this.thePlayer = this.worldObject.getClosestPlayerToEntity(this.theWolf, (double)this.minPlayerDistance);
+    public boolean shouldExecute() {
+        this.thePlayer = this.worldObject.getClosestPlayerToEntity(this.theWolf, (double) this.minPlayerDistance);
         return this.thePlayer == null ? false : this.hasPlayerGotBoneInHand(this.thePlayer);
     }
 
-    public boolean continueExecuting()
-    {
-        return !this.thePlayer.isEntityAlive() ? false : (this.theWolf.getDistanceSqToEntity(this.thePlayer) > (double)(this.minPlayerDistance * this.minPlayerDistance) ? false : this.field_75384_e > 0 && this.hasPlayerGotBoneInHand(this.thePlayer));
+    public boolean continueExecuting() {
+        return !this.thePlayer.isEntityAlive() ? false : (this.theWolf.getDistanceSqToEntity(this.thePlayer) > (double) (this.minPlayerDistance * this.minPlayerDistance) ? false : this.field_75384_e > 0 && this.hasPlayerGotBoneInHand(this.thePlayer));
     }
 
-    public void startExecuting()
-    {
+    public void startExecuting() {
         this.theWolf.func_70918_i(true);
         this.field_75384_e = 40 + this.theWolf.getRNG().nextInt(40);
     }
 
-    public void resetTask()
-    {
+    public void resetTask() {
         this.theWolf.func_70918_i(false);
         this.thePlayer = null;
     }
 
-    public void updateTask()
-    {
-        this.theWolf.getLookHelper().setLookPosition(this.thePlayer.posX, this.thePlayer.posY + (double)this.thePlayer.getEyeHeight(), this.thePlayer.posZ, 10.0F, (float)this.theWolf.getVerticalFaceSpeed());
+    public void updateTask() {
+        this.theWolf.getLookHelper().setLookPosition(this.thePlayer.posX, this.thePlayer.posY + (double) this.thePlayer.getEyeHeight(), this.thePlayer.posZ, 10.0F, (float) this.theWolf.getVerticalFaceSpeed());
         --this.field_75384_e;
     }
 
-    private boolean hasPlayerGotBoneInHand(EntityPlayer p_75382_1_)
-    {
+    private boolean hasPlayerGotBoneInHand(EntityPlayer p_75382_1_) {
         ItemStack itemstack = p_75382_1_.inventory.getCurrentItem();
         return itemstack == null ? false : (!this.theWolf.isTamed() && itemstack.getItem() == Items.bone ? true : this.theWolf.isBreedingItem(itemstack));
     }
-}
+}
